### PR TITLE
Gcp serverless vertex ai

### PR DESCRIPTION
**Pull request by mbukhori-gdp**
update from self managed to fully managed (serverless) GCP Vertex AI

example output : 
 <pre>
    Buitenzorg is the Dutch name for the city of Bogor, Indonesia.
    The name 'Buitenzorg' is derived from the Dutch words 'buiten' meaning 'outside' and 'zorg' meaning 'care' or 'worry'. It was named so because it was a place of relaxation and
    retreat for the Dutch colonial rulers, away from the worries and cares of their administrative duties in the capital city of Batavia (now Jakarta).

    During the Dutch colonial
    period, Buitenzorg was a popular hill station and resort town, known for its cool climate, beautiful gardens, and scenic views. The city was also home to the famous Buitenzorg Botanical Gardens, which were established in
    1817 and are still one of the largest and most important botanical gardens in Southeast Asia.

    After Indonesia gained independence in 1945, the city was renamed Bogor, which is the Sundanese word for 'fog' or
    'mist', reflecting the city's cool and misty climate. Today, Bogor is a thriving city and a popular tourist destination, known for its natural beauty, cultural attractions, and historical landmarks.

    Non-JSON response: data: [DONE]
    Python script executed successfully.</pre>

### CODE REVIEW

File: gcp-ai/.env.example [hunk 1]
```diff
@@ -3,12 +3,9 @@
...
 # Please refer to the following documentation for more details and guidelines:
 # Scope of Services: https://docs.google.com/document/d/1cXRjv34uXjluQzyRu027r5ax8GT-HOw3naMSPi8aeVs/edit?tab=t.0#heading=h.lfdykfqkf1d5 
-GCP_REGION="..."
-GCP_MACHINE_TYPE="..."
-GCP_ACCELERATOR_TYPE="..."
-GCP_ACCELERATOR_COUNT="..."
-GCP_VLLM_DOCKER_URI="..."
-GCP_MODEL_ID="..."
+GCP_REGION=us-central1
+GCP_ENDPOINT_NAME=... ##This value will be automatically filled after running the 1-click CLI.
```
Comment (muhammad-asn) [inline gcp-ai/.env.example:7]: Better diatas ini jgn disamping klo comment perlu isi value

**Review by muhammad-asn — APPROVED** [review]

File: gcp-ai/.env.example (hunk 1 above)
Comment (mbukhori-gdp) [reply gcp-ai/.env.example:7]: ohh keterangannya ye

Comment (mbukhori-gdp) [reply gcp-ai/.env.example]: updated bre @muhammad-asn

File: gcp-ai/setup_gcp_ai.sh [hunk 2]
```diff
@@ -144,31 +118,27 @@ else
...
+if [[ "$OS" == "Windows" ]]; then
+  source my_venv/Scripts/activate
+else
+  source my_venv/bin/activate
+fi
+
+# Clone the repository if not already present and navigate to the directory
 if [ ! -d "codehub" ]; then
-  git clone --filter=blob:none --sparse https://github.com/GDP-ADMIN/codehub.git
+  git clone --filter=blob:none --sparse git@github.com:mbukhori-gdp/GCP-AI.git codehub
```
Comment (muhammad-asn) [inline gcp-ai/setup_gcp_ai.sh:136]: Inu bener gini pak? Clonenya dari akun lau?

**Review by ignatiussuryowicaksono — APPROVED** [review]

File: gcp-ai/setup_gcp_ai.sh (hunk 2 above)
Comment (mbukhori-gdp) [reply gcp-ai/setup_gcp_ai.sh:136]: ehh lupa blum ganti wkwk

Comment (mbukhori-gdp) [reply gcp-ai/setup_gcp_ai.sh]: updated sam @muhammad-asn . thank yu

File: gcp-ai/.env.example [hunk 3]
```diff
@@ -36,4 +35,6 @@ GCP_MODEL_NAME="..."
 # ===============================
 # To obtain these values, please send a request to our Ticket GDPLabs asking for your GCP Service Account.
 # Email address for Ticket GDPLabs team: https://docs.google.com/document/d/1cXRjv34uXjluQzyRu027r5ax8GT-HOw3naMSPi8aeVs/edit?tab=t.0#heading=h.8fs6t79x8394 
-GOOGLE_APPLICATION_CREDENTIALS="..." # FileName
+
+##The path value for this key.json will be automatically filled in after running the 1-click CLI if the key.json already exists,
```
[the remark this answers is not in the document]
Comment (mbukhori-gdp) [reply gcp-ai/.env.example:39]: emm di prerequisites udah di add sih, perlu juga kah ? 
https://github.com/GDP-ADMIN/codehub/pull/27/files#diff-2417cccd51aab8959d49c09a8ffdc9c53c76f8bbf39360d3fb45ec4a66837a9cR12

[the remark this answers is not in the document]
Comment (mbukhori-gdp) [reply gcp-ai/.env.example:39]: ane update jadi sebuah keterangan aja biar refers ke prerequisites @darfisultoni

**Pull request merged**